### PR TITLE
Updated link to point to GoQuorum genesis file

### PR DESCRIPTION
**Pull request by siladu**
I think I have updated the link correctly which is an improvement. However, if you use [this genesis file](https://docs.goquorum.consensys.net/en/stable/HowTo/Configure/Consensus-Protocols/IBFT/#genesis-file) as your basis, then I think further changes are be required in order to correctly add a besu node into a goquorum network. For example, the documentation just says 
> update the consensus protocol specified in the config property from istanbul to ibft.

However, I'm not sure all the properties within `istanbul` are relevant to `ibft` in besu (although perhaps they can be safely ignored when you're not a validator). 
Also, `"isQuorum": true` yields an error in besu, so it either wants removing or changing to false, which would be worth documenting.

```
      "istanbul": {
        "epoch": 30000,
        "policy": 0,
        "ceil2Nby3Block": 0
      },
      "txnSizeLimit": 64,
      "maxCodeSize": 0,
      "isQuorum": true
```

Tagging @jframe who may be able to help clarify.

## Pull request checklist

Use the following list to make sure your PR fits the Besu doc quality standard.

### Before creating the pull request

Make sure that:

- [x] [all commits in this PR are signed off for the DCO](https://wiki.hyperledger.org/display/BESU/DCO).
- [x] you read the [contribution guidelines](https://wiki.hyperledger.org/display/BESU/Contributing+to+documentation).
- [x] you have [tested your changes locally](https://wiki.hyperledger.org/display/BESU/MkDocs+And+Markdown+Guide#MkDocsAndMarkdownGuide-PreviewTheDocumentation) before submitting them to the community for review.

### After creating your pull request and tests finished

Make sure that:

- [ ] you fixed all the issues raised by the tests, if any.
- [ ] you verified the rendering of your changes on [ReadTheDocs.org PR preview](https://wiki.hyperledger.org/display/BESU/MkDocs+And+Markdown+Guide#MkDocsAndMarkdownGuide-PreviewwithReadTheDocs)
  and updated the testing link (see [Testing](#testing)).

### For content changes

- [x] Doc content
- [ ] Doc pages organisation

## Testing

Link on https://besu.hyperledger.org/en/stable/HowTo/Configure/Consensus-Protocols/QuorumIBFT/ should point to https://docs.goquorum.consensys.net/en/stable/HowTo/Configure/Consensus-Protocols/IBFT/#genesis-file

### CODE REVIEW

**Review by alexandratran — APPROVED** [review]
Added note about removing `isQuorum`. If the `istanbul` properties can ignored and the node will properly be added as a non-validator, then lgtm.

File: docs/HowTo/Configure/Consensus-Protocols/QuorumIBFT.md [hunk 1]
```diff
@@ -9,8 +9,10 @@ Besu nodes can operate (that is, submit transactions and receive blocks) in a Qu
 
 To connect to a Quorum IBFT 1.0 network:
 
-1. In the Quorum IBFT 1.0 [genesis file](../Genesis-File.md), update the consensus protocol specified in the `config`
-   property from `istanbul` to `ibft`.
+1. In the Quorum IBFT 1.0 [genesis file](https://docs.goquorum.consensys.net/en/stable/HowTo/Configure/Consensus-Protocols/IBFT/#genesis-file):
+
+    - Update the consensus protocol specified in the `config` property from `istanbul` to `ibft`.
```
Comment (jframe) [inline docs/HowTo/Configure/Consensus-Protocols/QuorumIBFT.md:14]: There are further changes required unfortunately going from GoQuorum to Besu
* GoQuorum istanbul config property epoch becomes epochlength in Besu ibft config
* GoQuorum --istanbul.blockperiod cmd line option becomes the blockperiodseconds in Besu ibft config
* GoQuorum --istanbul.requesttimeout cmd line option needs to be converted from milliseconds to seconds and becomes requesttimeoutseconds in Besu ibft config
* Besu doesn't support sticky validator nodes. So a GoQuorum config with a policy of 1 cannot be used with Besu

Comment (alexandratran) [reply docs/HowTo/Configure/Consensus-Protocols/QuorumIBFT.md]: Added these changes. Can you re-review @jframe?

File: docs/HowTo/Configure/Consensus-Protocols/QuorumIBFT.md [hunk 2]
```diff
@@ -9,14 +9,26 @@ Besu nodes can operate (that is, submit transactions and receive blocks) in a Qu
...
-1. In the Quorum IBFT 1.0 [genesis file](../Genesis-File.md), update the consensus protocol specified in the `config`
-   property from `istanbul` to `ibft`.
+1. In the [Quorum IBFT 1.0 genesis file](https://docs.goquorum.consensys.net/en/stable/HowTo/Configure/Consensus-Protocols/IBFT/#genesis-file):
+
+    - Update the consensus protocol specified in the `config` item from `istanbul` to `ibft`.
+    - In the `ibft` item:
+        - Change `epoch` to `epochlength`.
+        - Add `blockperiodseconds`, the minimum block time in seconds.
+        - Add `requesttimeoutseconds`, the timeout for each consensus round before a round change, in seconds.
+        - Set `policy` to 0.
```
Comment (jframe) [inline docs/HowTo/Configure/Consensus-Protocols/QuorumIBFT.md:19]: We shouldn't include the policy field isn't supported in Besu. Besu always rotates validators.

Comment (alexandratran) [reply docs/HowTo/Configure/Consensus-Protocols/QuorumIBFT.md]: Fixed.

**Pull request merged**